man/fido_cred_set_authdata.3

```diff
@@ -392,6 +394,13 @@ The type of a credential may only be set once.
 Note that not all authenticators support COSE_RS256, COSE_ES384, or
 COSE_EDDSA.
 .Pp
+The
+.Fn fido_cred_append_type
+function is the same as
+.Fn fido_cred_set_type
+but can be called repeatedly to request one of multiple types, in order
```

Comment:
consider rephrasing it as `Unlike fido_cred_set_type, fido_cred_append_type may be called repeatedly to request multiple different types` to avoid confusion. we may also want to point out (not necessarily here) that `fido_cred_set_authdata()`, `fido_cred_set_authdata_raw()`, and `fido_dev_make_cred()` may now change the credential type returned by `fido_cred_type()`.

Reply:
yes, the part about where authdata updates the type may be surprising but at least it doesn't happen with `fido_cred_set_type()`, only `fido_cred_append_type()`.  however, the fact that the type changes may be enough reason to consider this breaking and warrant another function returning the `attcred.type` instead, but that may be equally confusing.